std_msgs/Header header
int64 ctl_mode                # ctl_mode reading
                              # to be described


